invenio_app_ils/ui/backoffice/src/common/api/base.js

```diff
@@ -24,4 +24,36 @@ export const http = axios.create({
     Authorization: `bearer ${token}`,
   },
   withCredentials: true,
-});
+};
+
+export const http = axios.create(apiConfig);
+
+const toIsoString = date => {
```

Comment:
Well, I have evaluated that already, but it is a NOGO for what we need:
* https://bundlephobia.com/result?p=moment@2.22.2
* https://github.com/moment/moment/issues/3376

Reply:
Actually, in my current develpoment (next PR) I am finding my self manipulating dates again, so it makes sense now to add a lib since there is nothing native that works nicely.
After [investigation](https://news.ycombinator.com/item?id=16912304), it looks like [Luxon](https://github.com/moment/luxon) (Memento child) is a good compromise. Here some ["why"](https://moment.github.io/luxon/docs/manual/why.html).
I am using an it looks quite nice. WDYT?

Reply:
date-fns could be a candidate, I haven't use it though so no strong opinions, here is a [comparison](https://github.com/date-fns/date-fns/issues/275#issuecomment-264934189) with moment.